app/src/main/res/layout/item_shop_management.xml

```diff
@@ -8,55 +8,72 @@
         android:layout_height="wrap_content"
-        android:orientation="horizontal"
-        android:padding="@dimen/dp_3"
+        android:layout_marginBottom="@dimen/dp_2"
+        android:background="@drawable/border_comment"
+        android:onClick="@{() -> viewModel.onItemClicked()}"
+        android:padding="@dimen/dp_2"
         >
-        <!--//todo image shop-->
-        <ImageView
-            android:layout_width="@dimen/dp_100"
-            android:layout_height="@dimen/dp_80"
-            android:layout_margin="@dimen/dp_2"
-            android:contentDescription="@string/shop"
-            />
-        <RelativeLayout
+        <LinearLayout
             android:layout_width="match_parent"
-            android:layout_height="@dimen/dp_80"
-            android:layout_marginStart="@dimen/dp_5"
+            android:layout_height="wrap_content"
+            android:layout_margin="@dimen/dp_1"
+            android:background="@color/light_gray"
+            android:orientation="horizontal"
             >
-            <TextView
-                android:id="@+id/text_shop_name"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_marginTop="@dimen/dp_10"
-                android:ems="8"
-                android:textSize="@dimen/sp_20"
-                />
-            <TextView
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_alignParentLeft="true"
-                android:layout_alignParentStart="true"
-                android:layout_below="@id/text_shop_name"
-                android:layout_marginTop="@dimen/dp_20"
-                android:ems="10"
-                android:textSize="@dimen/sp_15"
-                />
             <ImageView
-                android:layout_width="@dimen/dp_50"
-                android:layout_height="@dimen/dp_60"
-                android:layout_alignParentEnd="true"
-                android:layout_alignParentRight="true"
-                android:layout_alignTop="@+id/text_shop_name"
-                android:contentDescription="@string/shop_management"
-                app:srcCompat="@drawable/ic_next_fragment"
+                android:layout_width="@dimen/dp_100"
+                android:layout_height="match_parent"
+                android:layout_margin="@dimen/dp_1"
+                android:contentDescription="@string/title_shop"
+                android:scaleType="centerCrop"
+                app:imageUrl="@{viewModel.shopImage}"
                 />
-        </RelativeLayout>
-    </LinearLayout>
+            <RelativeLayout
+                android:layout_width="match_parent"
+                android:layout_height="@dimen/dp_80"
+                android:layout_marginLeft="@dimen/dp_5"
+                android:layout_marginStart="@dimen/dp_5"
+                >
+                <TextView
```

Comment:
WARNING: [RelativeOverlap] `@id/text_name_shop` can overlap `ImageView-5` if @{viewModel.shopName} grows due to localized text expansion
 If relative layout has text or button items aligned to left and right sides they can overlap each other due to localized text expansion unless they have mutual constraints like `toEndOf`/`toStartOf`.